Main.java

```diff
@@ -1,15 +1,44 @@
+import java.util.Scanner;
 /**
  *
- * @author 
+ * @author Sevde Korkut
  */
-public class Main {
+public class Main { 
 
   /**
    * The method that is executed when you hit the run button.
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // create a Scanner for user input 
+    Scanner input = new Scanner (System.in); 
+    System.out.println ("How many chromebook chargers?"); 
+  
+   // declare a variable to store chromebook
+   int chromebook = input.nextInt();
+   System.out.println ("How many motherboards?");
+   // declare a variable to store motherboards 
+   int motherboards = input.nextInt ();
+   
+   System.out.println ("How many mice?");
+   // declare a variable to store mice 
+   int mice = input.nextInt (); 
+   //declare a variable to store Subtotal 
+   double Subtotal = chromebook*34.99 + motherboards *127.50 + mice*18.00;
+
+   // show the subtotal
+   System.out.println("Subtotal = $" + Subtotal);
+   // declare a variable to store Taxes 
+   double Taxes = Subtotal*0.13; 
+   // show the taxes 
+   System.out.println("Taxes = $" + Taxes);
+   // declare a variable to store subtotal
+   double Total = Subtotal + Taxes;
+   //show the Total
+   System.out.println("Total = $" + Total);
```

Comment:
Good work. Remember that variable names should always begin with a lowercase letter though. So Subtotal should be subtotal, Taxes, should be taxes, etc. Otherwise, great work!